packages/@aws-cdk/aws-codedeploy/lib/ecs/deployment-group.ts

```diff
@@ -100,3 +368,90 @@ class ImportedEcsDeploymentGroup extends cdk.Resource implements IEcsDeploymentG
     this.deploymentConfig = props.deploymentConfig || EcsDeploymentConfig.ALL_AT_ONCE;
   }
 }
+
+/**
+ * {@link Duration} of infinite days
+ */
+//export const DurationInfinity = cdk.Duration.days(Number.POSITIVE_INFINITY);
+
+export class EcsBlueGreenDeploymentConfig {
+  static DurationInfinity = cdk.Duration.days(Number.POSITIVE_INFINITY);
+
+  /**
+   * The time to wait for a ContinueDeployment after the deployment is ready.
+   *
+   * @default - undefined - automatically continue once the deployment is ready.
+   */
+  readonly waitTimeForContinueDeployment?: cdk.Duration;
+  /**
+   * The time to wait before terminating old tasks.
+   * Set to {@link DurationInfinity} to keep the old tasks alive and deregistered from the load balancer.
+   *
+   * @default - undefined - terminate tasks immediately.
```

Comment:
This is different from the default you had above?  Feels confusing to have different defaults across the two interfaces

Reply:
Fixed.